osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -222,10 +223,17 @@ private void load(LocalisationManager localisation)
                         Direction = FillDirection.Vertical,
                         Padding = new MarginPadding { Top = 14, Right = shear_width / 2 },
                         AutoSizeAxes = Axes.Both,
-                        Children = new Drawable[]
+                        Children = new[]
                         {
+                            createStarRatingDisplay(beatmapInfo).With(display =>
+                            {
+                                display.Anchor = Anchor.CentreRight;
```

Comment:
Why not `TopRight`? Although it looks the same either way, it messes up the (x, y) coords:
![dotnet_F8tlegmHub](https://user-images.githubusercontent.com/35318437/91742517-3ca9a400-eb6b-11ea-8520-80c4796257cc.png)

I prefer this, as it starts at (0, 0).
![dotnet_eWiPP1vdP8](https://user-images.githubusercontent.com/35318437/91742535-429f8500-eb6b-11ea-9be6-2effb6931fee.png)